docs/install.md

```diff
@@ -18,10 +20,25 @@ LISA supports to run on Windows and Linux. Follow below steps to install LISA fr
 
 ## Install Python
 
-Install latest [Python 3.8 64 bits](https://www.python.org/). If there are Python installed already, please make sure effective Python's version is 3.8 64 bits or above.
+Lisa is tested on [Python 3.8 64 bits](https://www.python.org/). If there are Python installed already, please make sure effective Python's version is 3.8 64-bit or above.
 
```

Comment:
Wait, don't we use Poetry to stick to that Python version? Why have the user make sure of anything like that?

Reply:
If the user has multiple python version installed, then the python project s created with the version that installed poetry with. To ensure specific version, I think pyproject.toml needs to be updated with the desired python version and then need to run "poetry update"

Reply:
Now I get it: Poetry itself does *not* fetch the python interpreter, just python packages. Sigh, disappointing. OK, I'll stick to Python 3.9 on my end and be the bug reporter, for anything 3.8-specific.

Reply:
You are right, Poetry built on top of virtual env, it does manage Python packages only. So far, it's used to make sure we have same package minor versions. In future, it may be helpful on build our packages.